src/Symfony/Component/Notifier/Bridge/FortysixElks/FortysixElksTransport.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FortysixElks;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Jon Gotlin <jon@jon.se>
+ */
+class FortysixElksTransport extends AbstractTransport
+{
+    protected const HOST = 'api.46elks.com';
+
+    private string $apiUsername;
```

Comment:
username would be enough I guess

Reply:
I used the api-prefix to not confuse with username and password that is used to log in on the website. 46elks is also using the prefix in their api documentation.